app_check/src/attest/app_attest_provider_factory.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Google LLC
```

Comment:
I think this should be under: src/include/firebase/app_check

And either app_attest.h, or app_attest_provider.h

Same for the other default providers.